CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ numbering uses [semantic versioning](http://semver.org).
 
 NOTE: The change log until version `v0.2.4` is auto-generated.
 
+## [v1.4.0](https://github.com/Xtigyro/puppetserver-helm-chart/tree/v1.4.0) (2019-12-03)
```

Comment:
That should not be here. This has already been part of the `master` branch.

Reply:
Yes, I have rebased off of master a couple times.

Reply:
It looks like the issue is I have not rebased recently enough.  I will get that done and push updates again.

Reply:
Ah - I see. Thank you!

Reply:
I did rebase again and it did not help with this type of diff.  They just don't make sense.  They seem to claim that your master branch has not had these updates, which it clearly has.  I am tempted to create a new PR via a cleaner branch, starting with your current master...  at least to see what the PR looks like.

Reply:
My bigger concern is losing/messing up the current code. I don't care so much for the wrongly displayed diff. If there's a problem upon merging - the `master` branch will need a real rebase to revert properly to the prev state.